content/en/profiler/enabling/supported_versions.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Language and Tracer Versions for Profiler Features
+kind: documentation
+disable_sidebar: true
+further_reading:
+- link: "/profiler/enabling"
+  tag: "Documentation"
+  text: "Enabling Profiler"
+---
+
+The following table shows a summary of the features available for each language runtime. 
+- **Minimum versions** are the minimum required to get access to a particular feature-If you have an earlier version, that feature is not supported. 
+- **Recommended versions** are the runtime and tracer versions that give you access to **all** the supported features. Generally speaking, it's best if you update to the latest version of all tracers.
+
+More details about runtime support and profile types is available on each language's profiler setup page (click the language heading in the table to access).
+
+|                                   | [Java][1]  | [Python][2]  |  [Go][3]   |  [Ruby][4] |   [Node.js][5]  |  [.NET][6]   |   [PHP][7]  | [Rust/C/C++][8] |
+|-----------------------------------|:-------:|:-------:|:------------:|:------:|:---------:|:-------:|:------:|:----------:|
+| Operating systems                 | | | | Linux only | | | CentOS&nbsp;7+ | Linux&nbsp;v4.17+ |
+| Minimum&nbsp;runtime&nbsp;version | JDK&nbsp;8+ | Python&nbsp;2.7+ | Go&nbsp;1.20+ | Ruby&nbsp;2.3+ | Node&nbsp;14+ | .NET&nbsp;Core&nbsp;2.1+, .NET&nbsp;5+, .NET&nbsp;Framework&nbsp;4.6.1+ | PHP&nbsp;7.1+ |  |
+| Recommended runtime version | JDK&nbsp;11+ | Python&nbsp;3.6+ | Go&nbsp;1.21+ | Ruby&nbsp;2.3+ | Node&nbsp;14+ | .NET&nbsp;6+ | PHP&nbsp;7.1+ | |
+| Minimum tracer version   | | 0.35.0 | 1.23.0 | 0.48.0 | 0.23.0 | 2.7.0| | |
+| Recommended tracer version |    | 2.3.2  | 1.57.0 | 1.15.0 | 2.12.0 | 2.31.0  | 0.92.0 | |
```

Comment:
Isn't the recommended version always the latest?
I think we did not have this version number anywhere in the docs, and this would mean to update this section with every release from now one.

Reply:
I went ahead and just linked to the releases page in the repo so readers can quickly find the latest version without us needing to maintain this.